src/fideslang/models.py

```diff
@@ -20,7 +20,7 @@
     validator,
 )
 
-from fideslang.validation import (
+from fideslang.validation import (  # no_self_reference_in_data_flow,
```

Comment:
```suggestion
from fideslang.validation import (
```

I think this works to remove it?

Reply:
perfect, thanks!